.rubocop.yml

```diff
@@ -0,0 +1,3 @@
+AllCops:
+  SuggestExtensions: false
+  NewCops: disable
```

Comment:
enabled please

Reply:
Using SRP and rubocop to refactor the code and passing the github actions